config/routes.rb

```diff
@@ -7,7 +7,9 @@
 
   resources :courses do
     resources :posts do
+      resources :ratings
```

Comment:
I wonder if "member...end" can be used here too. It forces the route before to have a specific id.

Reply:
I'm not sure. If you think that'd be useful it's worth trying